File: 백준/Bronze/1934. 최소공배수/최소공배수.py
import sys

def gcd(a, b):
    while b:
        a, b = b, a % b
    return a

def lcm(a, b):
    return a * b // gcd(a, b)

T = int(sys.stdin.readline())
for _ in range(T):
    a, b = map(int, sys.stdin.readline().split())
    print(lcm(a, b))

File: 백준/Bronze/1934. 최소공배수/최소공배수.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
const T = input[0];
const numberList = input.slice(1);
const gcd = (a, b) => {
    if (b === 0) {
        return a;
    }
    return gcd(b, a % b);
};

numberList.forEach((list) => {
    const [a, b] = list.split(' ').map(Number);
    const g = gcd(a, b);
    const lcm = (a * b) / g; // 최소공배수 계산
    console.log(lcm);
});
